### PR TITLE
Removed API Supervisor

### PR DESCRIPTION
**Pull request by VedangW**
The functionality of API Supervisor was added to TS.Supervisor and the appropriate documentation was added. TS.Supervisor now spawns the worker child, API.

### CODE REVIEW

File: data/times.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+33,49,63,80,96,115,132,144,155,171,182,193,204,214,231
+109,191,275,315,356,395,433,463,505,541,573,606,634,663,697
+23,24,34,43,61,85,106,135,160,171,185,206,223,241,259
+85,124,177,239,324,387,397,405,414,423,431,441,448,456,467
```
Comment (prasadtalasila) [inline data/times.csv]: Why do we need this file? Is there a new test utilising this file?

File: lib/ts/supervisor.ex [hunk 2]
```diff
@@ -26,6 +26,12 @@ defmodule TS.Supervisor do
 
 	args
 
+	###Function of Supervisor
+	The module TS.Supervisor is the top level supervisor for the project. The 
+	children of a Supervisor can be workers or supervisors. TS.Supervisor spawns
+	three children, the Network Constructor (worker), the API (worker)
+	and a Station Supervisor (Supervisor). 
+
 	### Strategies
```
Comment (prasadtalasila) [inline lib/ts/supervisor.ex]: comments in the deleted file api_supervisor.ex are very helpful. They are generic and hence can be copied to supervisor.ex. Please do copy them.

File: data/times.csv (hunk 1 above)
Comment (VedangW) [inline data/times.csv]: No sir I haven't written any new tests for this file. However, there are at least two places (lines 68 and 80) in api.ex where this file is opened. Can you please clarify the role of this file in this context?

File: lib/ts/supervisor.ex (hunk 2 above)
Comment (VedangW) [inline lib/ts/supervisor.ex]: I will go through them once again and add the comments sir.

File: data/times.csv (hunk 1 above)
Comment (prasadtalasila) [inline data/times.csv]: I am not sure about this. Let @suhasspai answer the question.

But, if the file is needed and was not there in the repository before, how are the tests going through?

**Comment by prasadtalasila** [comment]
Please make the necessary changes and open a PR to the dev branch. I am closing this PR.

**Comment by prasadtalasila** [comment]
Required changes:

1. Copy extra comments from api_supervisor.ex to supervisor.ex
2. Remove data/times.csv and run ts application - data/times.csv must be generated again. If so, remove the existing file. Add the file to .gitignore.

**Pull request closed without merging**